src/sidebar/components/HypothesisApp.js

```diff
@@ -173,6 +173,7 @@ function HypothesisApp({ auth, bridge, settings, session, toastMessenger }) {
       {route !== 'notebook' && (
         <TopBar
           auth={authState}
+          bridge={bridge}
```

Comment:
Would it be preferable to inject the bridge service instead of passing from the HypothesisApp? I think we should be consistent among the components in the application.

Reply:
Oh indeed. The `bridge` prop is listed as an `injectedProp` at the bottom of the file so it is supposed to be injected as a service.